app/instances.go

```diff
@@ -159,18 +159,18 @@ func (a *App) createTriggers(tConfigs []*trigger.Config, runner action.Runner) (
 							var ok bool
 							ref, ok = support.GetAliasRef("action", act.Ref)
 							if !ok {
-								return nil, fmt.Errorf("action '%s' not imported", act.Ref)
+								return nil, fmt.Errorf("action '%s' not imported in trigger [%s]'s handler [%s]", act.Ref, tConfig.Id, hConfig.Name)
```

Comment:
fmt.Errorf("action '%s' referenced in trigger [%s]'s handler [%s], has not been imported"